unlock-app/src/theme/media.ts

```diff
@@ -1,7 +1,7 @@
 import styled, { css } from 'styled-components'
```

Comment:
Changing this file was a huge but necessary pain, thanks to TypeScript. Note that the types are not very concrete; at some point in the future this should be rectified.

Reply:
ok, that should make for easy splitting!